src/node_server_actions_1.cpp

```diff
@@ -251,7 +251,7 @@ node_count_type node_server::regrid(const hpx::id_type &root_gid, real omega, re
 	tstop = timer.elapsed();
 	printf("Formed tree in %f seconds\n", real(tstop - tstart));
 	printf("solving gravity\n");
-	solve_gravity(grav_energy_comp, !opts().output_filename.empty());
+	solve_gravity(grav_energy_comp, false);
```

Comment:
Can this change be explained?
This change can potentially affect all simulations.